postgres/postgres.go

```diff
@@ -840,6 +854,33 @@ func (adapter *Postgres) Update(SQL string, params ...interface{}) (sc adapters.
 		return
 	}
 	log.Debugln("generated SQL:", SQL, " parameters: ", params)
+	if strings.Contains(SQL, "RETURNING") {
```

Comment:
you can use row_to_json postgtres func instead build return on your own

Reply:
... if you knew the time I lost on that... I'll use row_to_json

Reply:
finally, I'm not so sure I wasted my time anymore, because I can't use row_to_json with an update, I'm supposed to use json_object or json_build_object, but then I have to declare the value of my keys before the request, which I can't do in the case of "*". My solution has the advantage of building the json object when the row returns, regardless of any supposed structures.